fastai/callback/schedule.py

```diff
@@ -218,7 +218,7 @@ def valley(lrs:list, losses:list, num_it:int):
     sections = (max_end - max_start) / 3
     idx = max_start + int(sections) + int(sections/2)
 
-    return lrs[idx], (lrs[idx], losses[idx])
+    return lrs[idx].item(), (lrs[idx].item(), losses[idx])
```

Comment:
I think this will fail if the suggestion isn't a tensor (which it might not be). Instead I think this should always work:

```suggestion
    return float(lrs[idx]), (float(lrs[idx]), losses[idx])
```

Reply:
Yes. Will use that and try to run the test on my computer before next step.